sdk/containerregistry/Microsoft.Azure.Management.ContainerRegistry/tests/Helpers/ContainerRegistryTestUtilities.cs

```diff
@@ -121,6 +133,51 @@ public static Replication CreatedContainerRegistryReplication(ContainerRegistryM
                 });
         }
 
+        public static ScopeMap CreatedContainerRegistryScopeMap(ContainerRegistryManagementClient client, string resourceGroupName, string registryName)
+        {
+            return client.ScopeMaps.Create(
+                resourceGroupName,
+                registryName,
+                TestUtilities.GenerateName("acrscopemap"),
+                DefaultScopeMapActions
+            );
+        }
+
+        public static Token CreatedContainerRegistryToken(ContainerRegistryManagementClient client, string resourceGroupName, string registryName, string scopeMapId)
+        {
+            return client.Tokens.Create(
+                resourceGroupName,
+                registryName,
+                TestUtilities.GenerateName("acrtoken"),
+                new Token(
+                    scopeMapId: scopeMapId
+                )
+            );
+        }
+
+        public static ConnectedRegistry CreatedContainerRegistryConnectedRegistry(ContainerRegistryManagementClient client, string resourceGroupName, string registryName, string connectedRegistryName, string tokenId)
+        {
+            return client.ConnectedRegistries.Create(
+                resourceGroupName,
+                registryName,
+                connectedRegistryName,
+                new ConnectedRegistry(
+                    mode: "ReadWrite",
+                    parent: new ParentProperties(
+                        syncProperties: new SyncProperties(
+                            tokenId: tokenId,
+                            schedule: "0 9 * * *",
+                            messageTtl: XmlConvert.ToTimeSpan("PT48H"),
```

Comment:
@AntonioVT `messageTtl` and `syncWindow` are string property types in ISO format. We can remove the `XmlConvert.ToTimeSpan` conversion.

Reply:
Autorest generates the .NET SDK with property type `SystemTimeSpan` since we have `"format": "duration"` annotation in our [swagger](https://github.com/Azure/azure-rest-api-specs/blob/3f78f3bc2d2546511853ad1931360ccdd34b0175/specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2021-12-01-preview/containerregistry.json#L3127) for these properties. This change LGTM.